package.json

```diff
@@ -36,14 +36,16 @@
 		"onCommand:cosmosDB.createMongoDatabase",
 		"onCommand:cosmosDB.createDocDBDatabase",
 		"onCommand:cosmosDB.createDocDBCollection",
+		"onCommand:cosmosDB.createDocDBDocument",
 		"onCommand:cosmosDB.connectMongoDB",
 		"onCommand:cosmosDB.deleteMongoDB",
 		"onCommand:cosmosDB.deleteDocDBDatabase",
 		"onCommand:cosmosDB.deleteDocDBCollection",
+		"onCommand:cosmosDB.deleteDocDBDocument",
 		"onCommand:cosmosDB.newMongoScrapbook",
-		"onCommand:cosmosDB.updateMongoDocuments",
+		"onCommand:cosmosDB.updateDocuments",
 		"onCommand:cosmosDB.openMongoCollection",
-		"onCommand:cosmosDB.openDocDBCollection",
```

Comment:
Removed this because a left-click on the collection would give you the documents. Also, we aren't allowing any operations on the collection anyway.